apps/ledger-live-desktop/src/renderer/screens/market/index.tsx

```diff
@@ -89,7 +89,7 @@ export default function Market() {
   useEffect(() => {
     const intervalId = setInterval(() => refetchData(marketCurrentPage ?? 1), refreshRate);
 
-    return () => clearInterval(intervalId);
+    return () => clearInterval(intervalId as unknown as number);
```

Comment:
Not related to the PR but the CI failed because of this: https://github.com/LedgerHQ/ledger-live/actions/runs/8805039576/job/24166776064#step:7:28